src/js/view/view.js

```diff
@@ -774,6 +775,7 @@ function View(_api, _model) {
         }
         if (_controls) {
             _controls.destroyInstream(_model);
+            _controls.controlbar.el.querySelector('.jw-slider-time').setAttribute('tabindex', '0');
```

Comment:
These changes should be done in controls.js `setupInstream` and `destroyInstream` methods. The view should never deal with `controlbar` directly. This logic belongs in controls, and not in code delivered player without controls.